R/drive_publish.R

```diff
@@ -1,9 +1,15 @@
-#' Publish Google Drive file.
+#' Publish Google Drive file
 #'
-#' @template file
+#' Publish a Google Drive file to the web. You can only publish "Google" files, such
+#' as Google Documents, Google Spreadsheets, or Google Slides.
+#'
+#' @seealso Wraps the `revisions.update` endpoint:
+#'   * <https://developers.google.com/drive/v3/reference/revisions/update>
+#'
+#' @template file-plural
 #' @param ... Name-value pairs to add to the API request body, for example
 #'   `publishAuto = FALSE` will ensure that each subsequent revision will not be
-#'   automatically published (default here is `publishAuto = TRUE`).
+#'   automatically published (default here is `publishAuto = TRUE` and `publishedOutsideDomain = TRUE`).
```

Comment:
Let's make the example feature our default, i.e. amplify what it means for `publishAuto = TRUE`.